core.ts

```diff
@@ -1395,10 +1415,10 @@ export declare namespace PinejsClientCoreFactory {
 				$string: string;
 				[index: string]: Filter;
 		  };
-	export type Lambda = {
+	export interface Lambda {
```

Comment:
This linting fix actually exposed some typings bugs which I then had to go through and fix